.github/workflows/govmomi-go-tests.yaml

```diff
@@ -44,6 +44,13 @@ jobs:
 
       - name: Run Tests
         env:
+          COUNT: 1
+          RACE_HISTORY_SIZE: 5
           TIMEOUT: 5m
           TEST_OPTS: ""
-        run: go test -timeout $TIMEOUT -count 1 -race -v $TEST_OPTS ./...
+        run: |
+          TEST_COUNT=$COUNT \
```

Comment:
Could we directly set TEST_COUNT as env to avoid this additional mapping?

Reply:
(same for the other vars)

Reply:
Ah, yeah, those *are* env vars, aren't they :) 

Sure, I can do that!

Reply:
Awesome!